translations/monero-core_it.ts

```diff
@@ -1101,7 +1101,7 @@
     <message>
         <location filename="../pages/Sign.qml" line="419"/>
         <source>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;                            Signing address &lt;font size=&apos;2&apos;&gt;  ( Paste in  or select from &lt;/font&gt; &lt;a href=&apos;#&apos;&gt;Address book&lt;/a&gt;&lt;font size=&apos;2&apos;&gt; )&lt;/font&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;style type=&apos;text/css&apos;&gt;a {text-decoration: none; color: #FF6C3C; font-size: 14px;}&lt;/style&gt;                            Indirizzo di firma &lt;font size=&apos;2&apos;&gt;  ( Incolla  oppure seleziona da &lt;/font&gt; &lt;una href=&apos;#&apos;&gt;Rubrica&lt;/a&gt;&lt;font size=&apos;2&apos;&gt; )&lt;/font&gt;</translation>
```

Comment:
"una" needs to go back to being "a" because it is part of the HTML - eg: `<a href=`.
In other words: "`&lt;una href`" is incorrect.
Should be: "`&lt;a href`"